boards/renesas/ek_ra8m1/ek_ra8m1.dts

```diff
@@ -144,3 +144,17 @@
 };
 
 mikrobus_serial: &uart3 {};
+
+&adc0 {
+	compatible = "renesas,ra-adc";
+	status = "okay";
+	pinctrl-0 = <&adc0_default>;
+	pinctrl-names = "default";
+};
+
+&adc1 {
```

Comment:
Zephyr doesn't usually enable too many devices by default.

Reply:
@duynguyenxa , let's check this point. In case there's no explicit requirement, let's keep minimum as one device enabled by default.

Reply:
We fixed it. Keep adc0 as default only. Thanks for your feedback.